apis/validator/beacon-node-oapi.yaml

```diff
@@ -290,6 +316,40 @@ components:
           format: uint64
           nullable: true
           description: "The slot in which a validator must propose a block, or `null` if block production is not required."
+    Validator:
+      type: object
+      properties:
+        validator_pubkey:
+          $ref: '#/components/schemas/pubkey'
+        withrawal_credentials:
+          type: string
+          format: bytes
+          pattern: "^0x[a-fA-F0-9]{64}$"
+          description: "Hash of withdrawal credentials"
+        effective_balance:
+          type: integer
+          format: int64
+          example: 32000000000
+          description: "Balance at stake in Gwei."
+        slashed:
+          type: boolean
+          description: "Was validator slashed (not longer active)."
+        activation_eligibility_epoch:
+          type: integer
+          format: uint64
+          description: "When criteria for activation were met."
+        activation_epoch:
+          type: integer
+          format: uint64
+          description: "Epoch when validator activated. Null if not activated"
```

Comment:
Why Null instead of `FAR_FUTURE_EPOCH` as the field is in state?

Reply:
should be far future epoch^^